app/src/components/SearchBar.jsx

```diff
@@ -38,20 +44,20 @@ function SearchBar({ placeholder, data }) {
 
                 />
                 <div className="searchIcon">
-                    {filteredData.length === 0 ? (
-                        <img src={SearchSvg} alt="home" />
+                    {wordEntered.length > 0 ? (
+                        <img src={closeSvg} alt="closeIcon" onClick={clearInput} />
                     ) : (
-                        <img src={closeSvg} alt="home" />
+                        <img src={SearchSvg} alt="searchIcon" />
                     )}
                 </div>
             </div>
             {filteredData.length != 0 && (
-                <div className="dataResult">
+                <div className="dataResult" key={artists.id}>
                     {filteredData.slice(0, 15).map((value, key) => {
                         return (
-                            <a className="dataItem" href={value.link} target="_blank">
-                                <p>{value.band} </p>
-                            </a>
+                            <div className="dataItem" href={value.link} target="_blank">
```

Comment:
do we want the app to open a new window when doing a search or to redirect to a page with the results? if its the former this is fine, otherwise _blank might not be what we want unless i've missed something in react specifically

Reply:
Nice catch. Den är borta då den inte fyller en funktion. Vi använder Link för att koppla vidare till sidan.